.eslintignore

```diff
@@ -1,4 +1,5 @@
 models/index.js
 node_modules
 public/js/*.js
-public/fontawesome/js/*.js
\ No newline at end of file
+public/fontawesome/js/*.js
+routes/*.js
```

Comment:
Remove this once the database has been merged. Lint errors were thrown due to db code not implemented in this branch.